src/components/Catchphrases/Catchphrases.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useState } from 'react';
+import './catchphrase.css';
+
+export default function Catchphrases({ setCatchphrases }) {
+  const [currentPhrase, setCurrentPhrase] = useState('');
+  const handleSavePhrase = () => {
+    setCatchphrases((prevState) => [currentPhrase, ...prevState]);
+    setCurrentPhrase('');
+  };
+
+  return (
+    <div>
+      <label className='displayContents'>
+          your catchphrases:
+        <input onChange={(e) => setCurrentPhrase(e.target.value)} />
```

Comment:
In order to properly reset the input, you will need to give it a value here, `value={currentPhrase}`